tests/cases/standalone/common/create/create_type_alias.result

```diff
@@ -36,7 +36,7 @@ SHOW CREATE TABLE data_types;
 |            |   "b" BOOLEAN NULL,                                        |
 |            |   "vb" VARBINARY NULL,                                     |
 |            |   "dt" DATE NULL,                                          |
-|            |   "dtt" DATETIME NULL,                                     |
+|            |   "dtt" TIMESTAMP(6) NULL,                                 |
```

Comment:
What if we add a datetime column? Maybe we need a test.

Reply:
I'll add a test case later.

Reply:
[ff3ba74](https://github.com/GreptimeTeam/greptimedb/pull/5506/commits/ff3ba7407446d385cf51d6be128a9b76db06518f)